GameOfLife/Controller/GameManager.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace GameOfLife
+{
+    /// <summary>
+    /// Management of the game.
+    /// </summary>
+    public class GameManager
```

Comment:
all classes need interfaces

Reply:
Changes have been made in Iteration 2 branch.